docs/usage.md

```diff
@@ -3,130 +3,441 @@
+
+### Integrate Created Records into Workflow
+
+- Utilize the newly created records within your workflow. The records can be used for tracking, analysis, and response actions, contributing to an efficient data management process.
+
+Creating records is a fundamental task for managing and tracking data within FortiSOAR. By following the steps outlined and leveraging the FortiAI assistant’s capabilities, users can effectively create and manage records to support their workflow and data management needs.
+
+>[!NOTE]
+>Ensure that the records created are accurate and complete to maintain effective data management and support your workflow.
+
+## Update Record
+
+Updating records within the FortiAI assistant involves modifying existing entries with new or updated information. This functionality ensures that records remain accurate and relevant by reflecting the latest data.
+
+### Use Case Example: Updating Incident Details
+
+A common use case is updating an incident record with new information, such as additional findings or changes in status. For instance, if new evidence is discovered during an investigation, updating the incident record with this information helps maintain an accurate and comprehensive record.
 
-### Generate playbook steps using FortiAI
+### Updating Records Using FortiAI Assistant
 
-Assume for the moment that you are a playbook designer who is required to build workflows to mitigate various types of threats. 
+1. **Initiate Record Update**:
+   - Issue a command to update a record with the new information. For example:
 
-  >**NOTE**: It is recommended to create smaller playbook blocks for easier review and minimal changes in the JSON template generated by FortiAI. You can then combine these small playbook blocks to create a larger playbook for your desired workflow.
+    >*Update the incident with ID [ALERT-ID] with the following details: [Updated details]*
 
-One use case to mitigate threats could require playbook steps that that extract indicators from an alert and enrich them using a threat intelligence tool such as VirusTotal.
+   - The assistant prompts you to provide the updated information for the record.
 
-Based on the reputation, the indicator reputation is updated to *Malicious*, if the reputation returned by VirusTotal is *Suspicious*; or the indicator reputation is updated to *Good* if the reputation is anything but *Suspicious*.
+2. **Provide Updated Information**:
+   - Specify the details to be updated in the record. For example:
 
-You may not be a seasoned playbook developer yet and may require some assistance with developing this playbook.
+    >*Update the alert status to "Resolved" and add a note: [Details of the note]*
 
-Use FortiAI to help with this task by clicking the **Playbook Generation** option in the FortiAI flyout.
+   - The assistant applies the changes to the record based on the provided information.
 
-Type your requirements, such as
+3. **Review and Confirm**:
```

Comment:
Add a note here that updating any text type of field will result in overwriting exiting field value.

Reply:
Added